frontend/package.json

```diff
@@ -11,8 +11,10 @@
   },
   "dependencies": {
     "axios": "^1.7.7",
+    "jwt-decode": "^4.0.0",
     "next": "14.2.14",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-modal": "^3.16.1"
```

Comment:
you should exclude both package.json and package-lock.json from your pr, the changes you bring aren't anything new and we shouldn't be overriding the old files for nothing.

Reply:
I just downloaded the package.json from the repo and just replaced it, I think there was an error when I initially ran npm install.